### DISCUSSION TITLE
[Request]: links that go outside of GH should open in new tab

### DISCUSSION THREAD

**Discussion by elliott-king** [community/community#40853] [Repositories]
### Select Topic Area

Product Feedback

### Body

This behavior of GitHub is inconsistent compared to other websites. When clicking a link, most sites will open a cross-origin link in a new tab, but a same-origin link will be opened in the current tab. GitHub always opens in the same tab. This leads to confusing behaviors:
- My PR has a link to my CI/CD pipeline. I open it, intending to compare to the PR's code. It opens in the same tab
- I open a link that opens a local Application (eg, Slack). It navigates to the Slack redirect page, then opens the app locally. I go back to my browser, and have trouble finding what I was doing (the tab formerly holding GH now holds the Slack redirect page).

This is a feature request to enable the same behavior as elsewhere. It may be odd that I am comparing you to the rest of the web, but this is a major website that I use more than many others, so it is noticeable that the behavior is different.

**Comment by ossanna16** [comment]
Hi there @elliott-king and welcome to our community! Thank you for the feedback!
